fhir/com.b2international.snowowl.fhir.rest/src/com/b2international/snowowl/fhir/rest/FhirMetadataController.java

```diff
@@ -78,11 +74,6 @@
  * 
  * @since 8.0.0
  */
-@Tag(description="CapabilityStatement", name = FhirApiConfig.CAPABILITY_STATEMENT, extensions = { 
```

Comment:
The Tag itself is required to categorize the controller and its endpoints into an API group, could you please add it back?